cmd/influx_inspect/export/export.go

```diff
@@ -254,20 +260,23 @@ func (cmd *Command) writeFull(mw io.Writer, w io.Writer) error {
 }
 
 func (cmd *Command) write() error {
-	// open our output file and create an output buffer
-	f, err := os.Create(cmd.out)
-	if err != nil {
-		return err
+	var w io.Writer
+	if cmd.out == "-" {
+		w = cmd.Stdout
+	} else {
+		// open our output file and create an output buffer
+		f, err := os.Create(cmd.out)
+		if err != nil {
+			return err
+		}
+		defer f.Close()
+		// Because calling (*os.File).Write is relatively expensive,
+		// and we don't *need* to sync to disk on every written line of export,
+		// use a sized buffered writer so that we only sync the file every megabyte.
+		bw := bufio.NewWriterSize(f, 1024*1024)
```

Comment:
Do we want to buffer when writing to stdout?

Reply:
I was wondering that, and went back and forth. Would it be better to buffer internally, or to let the user choose to buffer with a pipe program like `buffer`, or `mbuffer` or `pv`?  Maybe those aren't part of standard distros, though.  What do you think?

Reply:
I'd vote for consistency, especially since this seems to be a performance enhancement (golang/go#36619 suggests that unbuffered writes to `os.Stdout` are also slower than some users expect)